core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss

```diff
@@ -0,0 +1,69 @@
+@use "variables" as *;
+
+:host {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    padding-bottom: $spacing-7;
+
+    .dot-experiment-report-content {
+        padding: $spacing-3 $spacing-5 $spacing-5 $spacing-5;
+        row-gap: $spacing-3;
+        width: 100%;
+        overflow-y: scroll;
+        height: 100%;
+        background-color: $color-gray-100;
+
+        p {
+            font-size: $font-size-medium;
+            line-height: $line-height;
+        }
+
+        .container {
+            background-color: $white;
+            padding: $spacing-4;
+            border-radius: $border-radius-xl;
+            min-height: 500px;
+
+            .content {
+                width: $container-reports-max;
+            }
+        }
+    }
+}
+
+::ng-deep {
+    p-tag .p-tag {
```

Comment:
Can we make this global somehow? not sure how...

Reply:
Moved to _tag.scss with comments to separate from the default statuses.